chapter1.md

```diff
@@ -43,55 +37,63 @@ All abundances in the compiled datasets were counts of individuals.
 
-The Zipf (or power law) distribution was derived based on branching processes and was one of the best fitting distributions in a recent meta-analysis of SADs [@ulrich2010].
+The Zipf (or power law) distribution was derived based on both branching processes and as the outcome of the McGill and Collin's [-@mcgillcollins2003] spatial model. It was one of the best fitting distributions in a recent meta-analysis of SADs [@ulrich2010]. We use the discrete form of the distribution which is appropriate for fitting discrete abundance data [@white2008].
 
 ### Analysis
 
-Following current best practices for fitting distributions to data and evaluating their fit, we used maximum likelihood estimation to fit models to the data [@clark1999; @newman2005; @white2008] and likelihood-based model selection to compare the fits of the different models [@burnham2002; @edwards2007]. These general best practices have recently been affirmed as best practices for species abundance distributions [@connolly2014; @matthews2014].
+Following current best practices for fitting distributions to data and evaluating their fit, we used maximum likelihood estimation to fit models to the data [@clark1999; @newman2005; @white2008] and likelihood-based model selection to compare the fits of the different models [@burnham2002; @edwards2007]. These general best practices have recently been affirmed as best practices for species abundance distributions [@connolly2014; @matthews2014]. This requires that likelihoods can for the models can be solved for and therefore we excluded models that lack probability mass functions and therefore likelihoods (including all niche preemption models). While methods have been proposed for comparing models without probability mass functions in this context [@alroy2015], these methods have not been evaluated to determine how well they perform compared to the widely accepted likelihood based approaches.
 
 For model comparison we used corrected Akaike Information Criterion (AICc) weights to compare the fits of models while correcting for differences in the number of parameters and appropriately handling the small sample sizes (i.e., numbers of species) in some communities [@burnham2002]. The Poisson lognormal and the negative binomial each have two fitted parameters, while the log-series distribution and the Zipf distributions have one fitted parameter each. The model with the greatest AICc weight in each community was considered to be the best fitting model for that community. We also assessed the full distribution of AICc weights to evaluate the similarity of the fits of the different models.
 
 In addition to evaluating AICc of each model, we also examined the log-likelihood values of the models directly. We did this to assess the fit of the model while ignoring corrections for the number of parameters and the influence of similarities to other models in the set of candidate models.
```

Comment:
I initially liked this idea, but---if I understand correctly---I've changed my mind.

Likelihoods are hard to compare across data sets because they depend on the size of the data set and on the variance in abundance.